City-of-Helsinki/drupal-helfi-platform-config
# UHF-3313: Map translations

**Notice!**
To prevent conflicts in HELfi platform config, check this PR first: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/217


How to test with running instance:
- Make sure your instance is up and running
- Update the module
  - `composer require drupal/helfi_platform_config:dev-UHF-3313_map_translations`
- Run updates: `make drush-cim drush-updb drush-cr`
- Run translation import command in shell: `drush locale:check && drush locale:update`
- Go to any unit and check that the maps link to new tab states:
   - In English: Open the map in a new window
   - In Finnish: Avaa kartta uuteen ikkunaan
   - In Swedish: Öppna kartan i nytt fönster
